src/pages/Models/Details/_model-details.scss

```diff
@@ -188,23 +188,27 @@
     th,
     td {
       &:nth-child(1) {
-        width: 20%;
+        width: 15%;
       }
 
       &:nth-child(2) {
-        width: 19%;
+        width: 25%;
       }
 
       &:nth-child(3) {
-        width: 16%;
+        width: 10%;
       }
 
       &:nth-child(4) {
-        width: 25%;
+        width: 20%;
```

Comment:
AZ doesnt need so much space. Maybe drop this to 15% and give instance ID (nth(5)) 15% too?